onsset/onsset.py

```diff
@@ -1918,12 +2007,19 @@ def get_grid_lcoe(self, dist_adjusted, elecorder, additional_transformer, year,
                                elec_loop=elecorder,
                                additional_transformer=additional_transformer,
                                capacity_factor=grid_calc.capacity_factor,
-                               get_max_dist=get_max_dist)
+                               get_max_dist=get_max_dist,
+                               unmet_demand=self.df[SET_UNMET_DEMAND + "{}".format(year)],
+                               fuel_cost_settlement=self.df[SET_MG_DIESEL_FUEL + "{}".format(year)],
+                               mg_diesel_calc = mg_diesel_calc,
+                               genset_backup=True
```

Comment:
Need to be able to specify whether to include genset_backup already in the runner.py or Jupyter Notebook files, and then it gets passed down to here. 
Also:

- If I set _genset_backup=False_ the code breaks because there is a different number of output arguments from the _get_lcoe_ method. I think even if we include the diesel genset (genset_backup=True), we only need to return three outputs from the _get_lcoe_ method.
- It would be good in _runner.py_ to be able to choose to include CNSE, diesel_backup or neither, to ensure CNSE and diesel_backup are not used at the same time.